ngraph/core/include/ngraph/op/squeeze.hpp

```diff
@@ -39,6 +39,8 @@ namespace ngraph
 
                 virtual std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
+
+                bool is_dynamic() const;
```

Comment:
Do we need to have this method? I didn't find any usages.

Reply:
Since Squeeze is a fused op `void op::util::FusedOp::validate_and_infer_types()` was complaining about dynamic shapes.
I added this method to skip it since shape validation was calculated in the `pre_validate_and_infer_types`:
```
void op::util::FusedOp::validate_and_infer_types()
{
    NGRAPH_OP_SCOPE(util_FusedOp_validate_and_infer_types);
    pre_validate_and_infer_types();

    if (!can_decompose_with_partial_shapes() && is_dynamic())
    {
        return;
    }
```